crates/polars-sql/src/functions.rs

```diff
@@ -987,7 +987,7 @@ impl SQLFunctionVisitor<'_> {
             // Array functions
             // ----
             ArrayContains => self.visit_binary::<Expr>(|e, s| e.list().contains(s)),
-            ArrayGet => self.visit_binary(|e, i| e.list().get(i)),
+            ArrayGet => self.visit_binary(|e, i| e.list().get(i, false)),
```

Comment:
Also worth asking before considering merging: does this make sense for default behavior for SQL? I'm not as familiar with the polars sql workflow